src/views/AmalgamationSelection.vue

```diff
@@ -247,6 +286,17 @@ export default class AmalgamationSelection extends Vue {
       ]
     }
 
+    // For Horizontal and Vertical amalgamation, set current business as Holding company
+    if (type === AmalgamationTypes.HORIZONTAL || type === AmalgamationTypes.VERTICAL) {
+      draftAmalgamationApplication.filing.amalgamationApplication.amalgamatingBusinesses = [
+        {
+          type: AmlTypes.LEAR,
+          role: AmlRoles.HOLDING,
```

Comment:
@ketaki-deodhar I just had a chat with @vysakh-menon-aot and he informed me that he's using HOLDING and PRIMARY separately, so please update this to set the correct role.

Reply:
Just to confirm, based on UXpin:
HOLDING  for Vertical
PRIMARY for Horizontal
Is this correct?

Reply:
Yes, correct.